pgvectorscale/src/access_method/build.rs

```diff
@@ -150,6 +151,66 @@ pub unsafe extern "C" fn aminsert(
     false
 }
 
+#[cfg(any(feature = "pg13"))]
+#[pg_guard]
+pub unsafe extern "C" fn aminsert(
+    indexrel: pg_sys::Relation,
+    values: *mut pg_sys::Datum,
+    isnull: *mut bool,
+    heap_tid: pg_sys::ItemPointer,
+    heaprel: pg_sys::Relation,
+    _check_unique: pg_sys::IndexUniqueCheck,
+    _index_info: *mut pg_sys::IndexInfo,
+) -> bool {
```

Comment:
Could you make it so both the pg13 and the pg >13 call the same underlying function to do the work here - something like:
```rust
#[cfg(any(feature = "pg14", feature = "pg15", feature = "pg16"))]
#[pg_guard]
pub unsafe extern "C" fn aminsert(...) -> bool {
  aminsert_inner(...)
}
#[cfg(any(feature = "pg13"))]
#[pg_guard]
pub unsafe extern "C" fn aminsert(...) -> bool {
  aminsert_inner(...)
}
unsafe fn aminsert_inner(...) -> bool { ... }
```

Reply:
fixed